site/content/en/docs/manual/advanced/analytics-and-monitoring/manual-qa.md

```diff
@@ -0,0 +1,204 @@
+---
+title: 'Manual QA and Review'
+linkTitle: 'Manual QA'
+weight: 2
+description: 'Guidelines on evaluating annotation quality in CVAT manually'
+---
+
+In the demanding process of annotation, ensuring accuracy is paramount.
+
+CVAT introduces a specialized **Review mode**, designed to streamline the
+validation of annotations by pinpointing errors or discrepancies in annotation.
+
+> **Note**: The **Review mode** is not applicable for 3D tasks.
+
+See:
+
+- [Review and report issues: review only mode](#review-and-report-issues-review-only-mode)
+  - [Assigning reviewer](#assigning-reviewer)
+  - [Reporting issues](#reporting-issues)
+  - [Quick issue](#quick-issue)
+  - [Assigning corrector](#assigning-corrector)
+  - [Correcting reported issues](#correcting-reported-issues)
+- [Review and report issues: review and correct mode](#review-and-report-issues-review-and-correct-mode)
+- [Issues navigation and interface](#issues-navigation-and-interface)
+  - [Issues tab](#issues-tab)
+  - [Issues workspace](#issues-workspace)
+  - [Issues comments](#issues-comments)
+- [Manual QA complete video tutorial](#manual-qa-complete-video-tutorial)
+
+## Review and report issues: review only mode
+
+Review mode is a user interface (UI) setting where a specialized
+**Issue** tool is available. This tool allows you to identify
+and describe issues with objects or areas within the frame.
+
+> **Note:** While in review mode, all other tools will be hidden.
+
+**Review** mode screen looks like the following:
+
+![Review mode screen](/images/review_mode_screen.jpg)
+
+### Assigning reviewer
+
+> **Note**: Only project owners and maintainers can assign reviewers.
+
+To assign a reviewer to the job, do the following:
+
+1. Log in to the Owner or Maintainer account.
+2. (Optional) If the person you wish to assign as a reviewer
+   is not a member of **Organization**, you
+   need to [Invite this person to the **Organization**](/docs/manual/advanced/organization/#invite-members-into-organization).
+3. Click on the **Assignee** field and select the reviewer.
+4. From the **Stage** drop-down list, select **Validation**.
+
+   ![Assigning reviewer](/images/image194.jpg)
+
+### Reporting issues
+
+To report an issue, do the following:
+
+1. Log in to the reviewer's account.
+2. On the **Controls** sidebar, click **Open and issue** (![](/images/image195.jpg)).
+3. Click on the area of the frame where the issue is occurring,
+   and the **Issue report popup** will appear.
+
+   ![Issue report window](/images/issue_report.jpg)
+
+4. In the text field of the **Issue report popup**, enter the issue description.
+5. Click **Submit**.
+6. After finishing the review, save the work.
```

Comment:
That is not obligatory. You do not need to save the work in advance. 
Issues are created on the server immediately after submission